### ISSUE TITLE
Add Support for Symfony 5

### ISSUE THREAD

**Issue by mkoert** [yvoyer/domain-event#14]
```
Error: Class 'Symfony\Component\EventDispatcher\Event' not found

/var/www/project/vendor/star/domain-event/src/Ports/Symfony/EventAdapter.php:17
/var/www/project/vendor/symfony/error-handler/DebugClassLoader.php:337
/var/www/project/vendor/star/domain-event/src/Ports/Symfony/SymfonyPublisher.php:37
/var/www/project/vendor/star/domain-event/src/Ports/Symfony/SymfonyPublisher.php:75
/var/www/project/src/ProductInventory/Infrastructure/EventStore/ProductEventStore.php:40
/var/www/project/src/ProductInventory/Domain/Commands/Product/RegisterProductHandler.php:23
/var/www/project/src/Core/Application/Messaging/ClassMapCommands.php:29
/var/www/project/tests/Functional/Http/Fixtures/ApiFixture.php:38
/var/www/project/tests/Functional/Http/Fixtures/ApiFixture.php:29
/var/www/project/tests/Functional/Http/Controller/ProductShowTest.php:21

```

`use Symfony\Contracts\EventDispatcher\Event;`

**Comment by mkoert** [comment]
```
TypeError: Argument 1 passed to Symfony\Component\EventDispatcher\Debug\TraceableEventDispatcher::dispatch() must be an object, string given, called in /var/www/project/vendor/star/domain-event/src/Ports/Symfony/SymfonyPublisher.php on line 37

/var/www/project/vendor/symfony/event-dispatcher/Debug/TraceableEventDispatcher.php:131
/var/www/project/vendor/star/domain-event/src/Ports/Symfony/SymfonyPublisher.php:37
/var/www/project/vendor/star/domain-event/src/Ports/Symfony/SymfonyPublisher.php:75
/var/www/project/src/ProductInventory/Infrastructure/EventStore/ProductEventStore.php:40
/var/www/project/src/ProductInventory/Domain/Commands/Product/RegisterProductHandler.php:23
/var/www/project/src/Core/Application/Messaging/ClassMapCommands.php:29
/var/www/project/tests/Functional/Http/Fixtures/ApiFixture.php:38
/var/www/project/tests/Functional/Http/Fixtures/ApiFixture.php:29
/var/www/project/tests/Functional/Http/Controller/ProductShowTest.php:21
```

**Comment by yvoyer** [comment]
fixed in #17
